code/modules/admin/verbs/machine_upgrade.dm

```diff
@@ -1,5 +1,13 @@
-ADMIN_VERB(machine_upgrade, R_DEBUG, "Tweak Component Ratings", ADMIN_VERB_NO_DESCRIPTION, ADMIN_CATEGORY_HIDDEN, obj/machinery/machine in world)
```

Comment:
Same thing here, but the define should be changed to ADMIN_VERB_AND_CONTEXT_MENU